src/components/Note/NoteReactionList/NoteReactionList.tsx

```diff
@@ -45,6 +45,9 @@ export const NoteReactionList = (props: NoteReactionListProps) => {
   const others = useAppSelector((state) => state.participants?.others) ?? [];
   const participants = [me, ...others];
 
+  const isModerator = useAppSelector((state) => ["OWNER", "MODERATOR"].some((role) => state.participants!.self.role === role));
+  const boardLocked = useAppSelector((state) => !state.board.data!.allowEditing);
```

Comment:
I'm not sure about negating the state for every use – would it not make more sense to just rename the `allowEditing` property to `boardLocked` or `isBoardLocked`?

Reply:
This would of course require a backend change as well, so this should be talked about beforehand.

Reply:
see #4145